src/lib/src/hal/remote/halremote.spec.ts

```diff
@@ -1,26 +1,37 @@
-import { Http, Response, ResponseOptions, RequestOptions } from '@angular/http';
-import { MockBackend } from '@angular/http/testing';
+import { TestBed } from '@angular/core/testing';
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
 import { Observable } from 'rxjs/Observable';
 import { ReplaySubject } from 'rxjs/ReplaySubject';
 import 'rxjs/add/observable/empty';
 import 'rxjs/add/operator/delay';
 
 import { HalRemote } from './halremote';
+import { HalLink } from '../doc/hallink';
 
 describe('HalRemote', () => {
 
-  const mockHttp = new Http(new MockBackend(), new RequestOptions());
-  const mockResponse = (data = {}, status = 200) => {
+  let mockHttp: HttpTestingController;
+  let httpClient: HttpClient;
+  /*const mockResponse = (data = {}, status = 200) => {
```

Comment:
can we just delete this rather than commenting it out?

Reply:
oops, yes!